src/components/MessageComposer.js

```diff
@@ -89,7 +92,23 @@ const StyledMessageComposer = styled.div`
   .flex {
     flex: 1;
   }
+
 `;
+const StyledMessageComposerSendButton = styled.button`
+color: ${theme.color.footerText};
+border:none
+background-color:transparent;
+&:hover{
```

Comment:
If the background is set to transparent then &:hover is redundant because there is no change

Reply:
its a whole different background with the button because we put the icon instead